front/pages/auth/signup/index.vue

```diff
@@ -70,3 +99,169 @@ export default {
   }
 }
 </script>
+
+<style lang="scss">
```

Comment:
use scoped style if it supposed to be used only on this page

Reply:
Yaa, thanks for the input. I will use scope on all the pages.

Reply:
I've found that if I use scope, some styles are not being applied like i am not able to give color to icons. But On each page I have specified styles inside #page_name element. So that will not conflict on any other page, that i have made sure.